pom.xml

```diff
@@ -128,6 +131,23 @@ THE SOFTWARE.
           </exclusion>
         </exclusions> 
       </dependency>
+      <dependency>
+	    <groupId>org.apache.ant</groupId>
+	    <artifactId>ant</artifactId>
+	    <version>${ant.version}</version>
+	  </dependency>
+      <dependency>
+	    <groupId>commons-io</groupId>
```

Comment:
I don't see any usages of commons-io in the changed code, why is the dependency necessary now?

Reply:
Same reason than ant, but not needed anymore after a new update